### PR TITLE
Use protobuf files from common repo

### PR DESCRIPTION
**Pull request by deadeyegoodwin**

### CODE REVIEW

**Review by GuanLuo — DISMISSED** [review]

File: build/server/CMakeLists.txt [hunk 1]
```diff
@@ -72,7 +72,13 @@ FetchContent_Declare(
   GIT_REPOSITORY https://github.com/triton-inference-server/backend.git
   GIT_TAG ${TRITON_BACKEND_REPO_TAG}
   GIT_SHALLOW ON
-)
+  )
```
Comment (CoderHam) [inline build/server/CMakeLists.txt:75]: Remove extra spacing before ')'

File: src/backends/backend/CMakeLists.txt [hunk 2]
```diff
@@ -66,6 +65,8 @@ endif() # TRITON_ENABLE_METRICS
 
 target_link_libraries(
   triton-backend-library
+  PRIVATE
+    proto-library
```
Comment (CoderHam) [inline src/backends/backend/CMakeLists.txt:69]: Add comment # repo-common just like we do for  L71/72. Also in the other files

File: src/test/custom/sdk/CMakeLists.txt [hunk 3]
```diff
@@ -75,12 +77,12 @@ install(
 
 # Install the triton and protobuf headers needed to build a custom
 # backend
-install(
-  FILES
-    ../../../core/model_config.h
-    ${CMAKE_CURRENT_BINARY_DIR}/../../../core/model_config.pb.h
-  DESTINATION include/src/core
-)
+#install(
```
Comment (CoderHam) [inline src/test/custom/sdk/CMakeLists.txt:80]: Did you mean to remove this?

**Review by CoderHam — CHANGES_REQUESTED** [review]
I noticed the jetson build failed for this pipeline. Around test-util build I believe:

```
fatal error: model_config.pb.h: No such file or directory
 #include "model_config.pb.h"
          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
````

**Comment by CoderHam** [comment]
Some tests require the proto files: L0_simple_go_client and L0_client_java
They copy these over from src/core in the Docker QA and will likely need to be modified for the new location of the proto files.

**Pull request merged**